waiter/src/waiter/process_request.clj

```diff
@@ -533,81 +517,68 @@
-                        (when waiter-debug-enabled?
-                          (swap! response-headers merge (instance->debug-headers instance prepend-waiter-url)))
-                        (confirm-live-connection-without-abort)
-                        (let [endpoint (request->endpoint request waiter-headers)
-                              {:keys [error] :as response}
-                              (metrics/with-timer!
-                                (metrics/service-timer service-id "backend-response")
-                                (fn [nanos]
-                                  (add-debug-header-into-response! "X-Waiter-Backend-Response-ns" nanos)
-                                  (statsd/histo! metric-group "backend_response" nanos))
-                                (async/<!
-                                  (make-request-fn instance request instance-request-properties
-                                                   passthrough-headers endpoint metric-group)))
-                              request-abort-callback (request-abort-callback-factory response)
-                              confirm-live-connection-with-abort (confirm-live-connection-factory request-abort-callback)]
-                          (when error
-                            (throw-response-error error reservation-status-promise instance response-headers))
-                          (process-backend-response-fn local-usage-agent instance-request-properties descriptor instance request
-                                                       reason-map response-headers reservation-status-promise
-                                                       confirm-live-connection-with-abort request-state-chan response))
-                        (catch Exception e
-                          ; A :client-error or :instance-error may already be in the channel in which case our :generic-error will be ignored.
-                          (deliver reservation-status-promise :generic-error)
-                          ; close request-state-chan to mark the request as finished
-                          (async/close! request-state-chan)
-                          ; Handle error lower down
-                          (throw e))))))
-                (catch Exception e
-                  ; Handle error lower down
-                  (throw (ex-info "Rethrowing exception from process" {:descriptor descriptor :source :pr/process} e)))))
-            (catch Exception e
-              (let [{:keys [descriptor source]} (ex-data e)
-                    exception (if (= source :pr/process) (.getCause e) e)]
-                (-> (process-exception-fn track-process-error-metrics request exception)
-                    (update :headers (fn [headers]
-                                       (merge @response-headers headers))))))))))))
+          (let [{:keys [service-id service-description]} descriptor
```

Comment:
Removed a call to `confirm-live-connection-without-abort` and the surrounding `try`/`catch` block.